app/javascript/common/API.js

```diff
@@ -35,4 +35,4 @@ export default {
   }
 };
 
-export const globalMockMode = false;
+export const globalMockMode = true;
```

Comment:
Oops?

Reply:
let's leave it true until Monday/Tuesday next week when APIs are ready... ;)